go.mod

```diff
@@ -26,10 +26,10 @@ require (
 	gomodules.xyz/jsonpatch/v2 v2.4.0
 	google.golang.org/api v0.151.0
 	google.golang.org/protobuf v1.33.0
-	istio.io/api v1.19.4
+	istio.io/api v1.21.5
 	istio.io/client-go v1.19.4
-	k8s.io/api v0.28.4
-	k8s.io/apimachinery v0.28.4
+	k8s.io/api v0.29.0
+	k8s.io/apimachinery v0.29.0
```

Comment:
These are also bumped. Was there a reason?

Reply:
left a comment in the upstream PR.